qiskit/qobj/__init__.py

```diff
@@ -64,6 +66,9 @@
 
 import warnings
 
+from qiskit.qobj.common import QobjExperimentHeader
+from qiskit.qobj.common import QobjHeader
```

Comment:
Why is this imported in a single line `from qiskit.qobj.common import QobjExperimentHeader, QobjHeader`?

Reply:
idk, just following the pattern that exists already in this file